mutmut/__main__.py

```diff
@@ -683,6 +687,8 @@ def new_tests(self):
 class PytestRunner(TestRunner):
     def execute_pytest(self, params, **kwargs):
         import pytest
+        params+=["--rootdir=.","--inline-snapshot=disable"]
```

Comment:
Hmm. This seems a bit weird. Do we need some config option in mutmut for this?

Reply:
maybe, I think it should not be required and it could also be a problem in inline-snapshot. But this is currently useful to make things work

Reply:
The --rootdir was important. I had several problems to convince mutmut to test the code under `mutants/`